src/nft/components/explore/WelcomeModal.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import Modal from 'components/Modal'
+import { useState } from 'react'
+import { X } from 'react-feather'
+import styled, { useTheme } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const Container = styled.div`
+  position: relative;
+  display: flex;
+  padding: 30% 24px 24px;
+  overflow: hidden;
+  height: fit-content;
```

Comment:
I am not particularly happy to set height as well as paddings in this case, but `fit-content` together with `overflow: hidden` was required to hide an overflow caused by an absolute background image. 

Sure, we could've just used a CSS background, but an equivalent of `src-set` for CSS backgrounds is still not widely supported.